### PR TITLE
Organic interface items rewrite and bug fixes

### PR DESCRIPTION
**Pull request by jupyterkat**
## About The Pull Request

RING A DING DING BABY I LOST THE BET

overall cleanup of the organic interface items

## How This Contributes To The Skyrat Roleplay Experience

0e2

## Proof of Testing

works
  
</details>

## Changelog
:cl:
fix: fixed vibrators turning invisible on use
fix: fixed toys not applying mood events on certain traits
fix: fixed grammar on toys
fix: toys now show that they are customizable and how to do it when available
fix: toys interactions panel is no longer hidden under a scroll, and is now a separate component
:cl:

### CODE REVIEW

**Comment by SpaceLoveSs13** [comment]
Is this the Sex 2.0?

**Comment by jupyterkat** [comment]
> Is this the Sex 2.0?

yes

File: modular_skyrat/modules/modular_items/lewd_items/code/lewd_items/fleshlight.dm [hunk 1]
```diff
@@ -54,30 +59,33 @@
 	. = ..()
 	if(!istype(target))
 		return
+	if(target.stat == DEAD)
```
Comment (SpaceLoveSs13) [inline modular_skyrat/modules/modular_items/lewd_items/code/lewd_items/fleshlight.dm:62]: Did- someone use toys on a dead body that necessitated this check?

**Review by SpaceLoveSs13 — APPROVED** [review]
Code looks good otherwise! I will run few test merges just to ensure the tgui is all good.

File: modular_skyrat/modules/modular_items/lewd_items/code/lewd_items/fleshlight.dm (hunk 1 above)
Comment (jupyterkat) [reply modular_skyrat/modules/modular_items/lewd_items/code/lewd_items/fleshlight.dm:62]: F_PstTAWgAALkS9

**Pull request merged**